# Request
---
# Response

float64[] xBlocks
float64[] phiBlocks
string[] nameBlocks
